selenium/pom.xml

```diff
@@ -118,7 +118,7 @@
         <browserstack.automateKey>hp3vg22JUprFFZzESJrz</browserstack.automateKey>
         <browserstack.username>developers44</browserstack.username>
         <browserstack.local>false</browserstack.local>
-        <aodnPortal>https://portal-systest.aodn.org.au/</aodnPortal>
+        <aodnPortal>https://portal-edge.aodn.org.au/</aodnPortal>
```

Comment:
@jonescc  This can be overridden as runtime parameter. Our default environment will be edge instead of systest and therefore I updated the config.